### ISSUE TITLE
Replacement variables suggestions not accessible with Safari + VoicOver

### ISSUE THREAD

**Issue by afercia** [Yoast/yoast-components#623]
Just a reminder to keep track of the issue. The options in the suggestions list are not announced by Safari + VoiceOver. This is an issue that should be solved upstream, as the draft.js editor doesn't allow `aria-owns` on the editor. There's an issue for this, see https://github.com/facebook/draft-js/issues/1736 and we're waiting for some reply since 2 months. 

Allowing `aria-owns`, as required by the ARIA 1.0 pattern, would make it work also with Safari + VoiceOver.

Currently, nothing is announced when navigating the options with the arrow keys:

Safari + VoiceOver

![screen shot 2018-06-27 at 12 38 28](https://user-images.githubusercontent.com/1682452/41973379-19243b4a-7a15-11e8-9135-48cba1358868.png)

Other browser / screen reader combinations are less strict and do announce the options:

Firefox 61 + NVDA

![screenshot 16](https://user-images.githubusercontent.com/1682452/41973407-30c833be-7a15-11e8-95a5-8aa0efdd7fe1.png)

If there's no progress upstream, maybe we should investigate if there's a way to pass an aria-owns attribute to the editor and an ID to the suggestions list.
